packages/draw/src/artist.ts

```diff
@@ -21,80 +18,54 @@ export class Artist {
       this.canvasInitialHeight,
     );
 
-    this.simulation = new Simulation({
-      data,
-      simulationConfig,
-      styles: this.styles,
-    });
-
-    this.click_map_colors = generateUniqueColors(data.nodes.length);
-    this.click_map_canvas = new Canvas();
-    this.visual_canvas = undefined;
-
-    this.zoomer = new Zoomer();
-    this.event_listeners = [];
-  }
-
-  public draw(): void {
-    if (isSSR()) return;
-
     this.visual_canvas = new Canvas(
       this.canvasElement,
       this.styles.deviceScale,
     );
 
+    this.zoomer = new Zoomer();
```

Comment:
Glad to see the class name `Zoomer` has survived this refactor haha

Reply:
I'm not going to lie, the thought of renaming it to Camera crossed my mind... but I realize it was the wrong call hahah